Library/Homebrew/cmd/which.rb

```diff
@@ -0,0 +1,17 @@
+module Homebrew
+  def which
+    cmd = ARGV.first
+
+    if HOMEBREW_INTERNAL_COMMAND_ALIASES.has_key? cmd
+      cmd = HOMEBREW_INTERNAL_COMMAND_ALIASES[cmd]
+    end
+
+    if (path = HOMEBREW_LIBRARY_PATH/"cmd/#{cmd}.rb"; path.file?)
```

Comment:
Yes, I just wonder if people will use the tool for internal commands. But it makes sense :smile: (I need more :coffee: )